### ISSUE TITLE
Error when doing yarn

### ISSUE THREAD

**Issue by talmax1124** [frappe/books#390]
error /Users/carlitos/books/node_modules/sqlite3: Command failed.****

This is my log: 
~/books master*
❯ yarn
yarn install v1.22.18
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
[5/5] 🔨  Building fresh packages...
[-/8] ⠠ waiting...
[7/8] ⠠ electron
[3/8] ⠠ sqlite3
[4/8] ⠠ fsevents
error /Users/carlitos/books/node_modules/sqlite3: Command failed.
Exit code: 1
Command: node-gyp rebuild
Arguments: 
Directory: /Users/carlitos/books/node_modules/sqlite3
Output:
gyp info it worked if it ends with ok
gyp info using node-gyp@8.4.1
gyp info using node@16.14.2 | darwin | arm64
gyp info find Python using Python version 3.9.12 found at "/opt/homebrew/opt/python@3.9/bin/python3.9"
gyp info spawn /opt/homebrew/opt/python@3.9/bin/python3.9
gyp info spawn args [
gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/carlitos/books/node_modules/sqlite3/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/usr/local/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/carlitos/Library/Caches/node-gyp/16.14.2/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/carlitos/Library/Caches/node-gyp/16.14.2',
gyp info spawn args   '-Dnode_gyp_dir=/usr/local/lib/node_modules/npm/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Users/carlitos/Library/Caches/node-gyp/16.14.2/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Users/carlitos/books/node_modules/sqlite3',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  CC(target) Release/obj.target/nothing/../node-addon-api/nothing.o
  LIBTOOL-STATIC Release/nothing.a
warning: /Library/Developer/CommandLineTools/usr/bin/libtool: archive library: Release/nothing.a the table of contents is empty (no object file members in the library define global symbols)
  ACTION deps_sqlite3_gyp_action_before_build_target_unpack_sqlite_dep Release/obj/gen/sqlite-autoconf-3360000/sqlite3.c
/bin/sh: python: command not found
make: *** [Release/obj/gen/sqlite-autoconf-3360000/sqlite3.c] Error 127
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (node:events:526:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)
gyp ERR! System Darwin 21.4.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/carlitos/books/node_modules/sqlite3

~/books master*
❯

**Comment by 18alantom** [comment]
Can you check with node 16.13.1?

p.s. the build process and sqlite3 source have been changed in this [PR](https://github.com/frappe/books/pull/381) which will soon be merged, do check if `yarn` works for you on this.

**Comment by talmax1124** [comment]
@18alantom Would you like me to clone this commit to see if it works like that?

**Comment by 18alantom** [comment]
Just the commit where those changes are made may not work. You'll have to checkout the entire PR cause of the switch from `sqlite3` to `better-sqlite3`.

Also, if you have a db I'd suggest making a copy of it cause I haven't pushed a patch yet. The PR is mostly done and should be merged before friday.

**Comment by 18alantom** [comment]
@talmax1124 I've merged the PR, let me know if `yarn` runs for you now.

**Comment by 18alantom** [comment]
Closing this cause no response, if any issue is still being faced, feel free to reply here or to create a new issue.
